test/integration/driver.go

```diff
@@ -48,8 +48,8 @@ const (
 	// budget is the maximum number of retries with 1s delays
 	budget = 90
 
-	// Mixer SHA *update manually*
-	mixerSHA = "ea3a8d3e2feb9f06256f92cda5194cc1ea6b599e"
+	// Mixer tag *update manually*
+	mixerTag = "demo"
```

Comment:
@rshriram We can call it "stable" only if we can apply that label in docker hub. From what I could see, it is not possible, see my previous comment.